Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+class Program
+{
+    static void Main()
+    {
+        Console.WriteLine("--------------------------\n\n Welcome to Calculator \n\n--------------------------\n");
```

Comment:
Please create a static class like "ApplicationMessages" that will contain all of your static read-only strings.

Reply:
Done.